ci-operator/config/openshift/cluster-authentication-operator/openshift-cluster-authentication-operator-release-4.20__periodics.yaml

```diff
@@ -0,0 +1,142 @@
+base_images:
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  ocp_4.20_base-rhel9:
+    name: "4.20"
+    namespace: ocp
+    tag: base-rhel9
+  ocp_builder_rhel-9-golang-1.24-openshift-4.20:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.24-openshift-4.20
+binary_build_commands: make build --warn-undefined-variables
+build_root:
+  from_repository: true
+images:
+- dockerfile_path: Dockerfile.rhel7
+  inputs:
+    ocp_4.20_base-rhel9:
+      as:
+      - registry.ci.openshift.org/ocp/4.20:base-rhel9
+    ocp_builder_rhel-9-golang-1.24-openshift-4.20:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.24-openshift-4.20
+  to: cluster-authentication-operator
+promotion:
+  to:
+  - disabled: true
+    name: "4.20"
+    namespace: ocp
+releases:
+  initial:
+    integration:
+      name: "4.20"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.20"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-aws-external-oidc
+  interval: 24h
+  steps:
+    cluster_profile: aws-3
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+      OPENSHIFT_SKIP_EXTERNAL_TESTS: "True"
+      TEST_SUITE: openshift/auth/external-oidc
+    test:
+    - ref: openshift-e2e-test-8h
+    workflow: openshift-e2e-aws
+  timeout: 8h0m0s
```

Comment:
For now, setting this to the highest possible timeout because they are long running jobs. That being said, it probably makes sense to consider splitting these tests up where possible so we can have much shorter runtimes for the set of tests.